ktor-shared/ktor-websockets/common/src/io/ktor/websocket/DefaultWebSocketSession.kt

```diff
@@ -46,18 +50,26 @@ public interface DefaultWebSocketSession : WebSocketSession {
      */
     @InternalAPI
     public fun start(negotiatedExtensions: List<WebSocketExtension<*>> = emptyList())
+
+    public companion object {
+        /**
+         * Ping interval meaning pinger is disabled.
+         * @see pingIntervalMillis
+         */
+        public const val PINGER_DISABLED: Long = 0
```

Comment:
To be discussed: With this constant code seems a bit clearer, but I'm not sure if I chose the right place for it.

Reply:
I think it'd be best under the `WebSockets` companion since that's the only type users are generally exposed to.

Reply:
The only thing I don't like is that it will be two constants, one for a client plugin and another for a server plugin.
I thought about making these constants pointing to the same constant in the shared module, but it will result in imports ambiguity.

```kotlin
public const val PINGER_DISABLED: Long = DefaultWebSocketSession.PINGER_DISABLED
```

Reply:
@bjhham, what do you think?

Reply:
Oh I see, maybe in that case it can be a top-level declaration.